src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/PrisonerReceivedDueToAdmissionEventTest.kt

```diff
@@ -0,0 +1,233 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.messaging
```

Comment:
This and the next test class look like new test classes; but actually all I've done is split them out of the original `PrisonerReceivedEventTest` (which now looks like a deletion!) because it was getting a bit big!
This test class (to do with prisoner admissions) has some new tests for the new functionality added in this PR